Source/Urho3D/CSharp/Managed/Graphics/IndexBuffer.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Urho3DNet
+{
+    public partial class IndexBuffer
+    {
+        unsafe public bool SetData(short[] indexData)
+        {
+            if (indexData == null || indexData.Length == 0) return false;
```

Comment:
Style. New line before `return` needed.

Reply:
Now i that think of it in case of `indexData == null || indexData.Length == 0` we should still call `SetData(IntPtr.Zero)`. It would return `false` but would also warn user about incorrectly used API. Now C# wrapper eats possible user error without giving indication that something went wrong. Same for `VertexBuffer`.

Reply:
size and nullity is taken care of in C++, hence, no need for check in C#